interface/backend/api/views.py

```diff
@@ -22,22 +22,27 @@
 from ..cases import enums
 
 
-class CaseViewSet(viewsets.ModelViewSet):
+class ViewSetBase(viewsets.ModelViewSet):
+    def get_serializer_context(self):
+        return {'request': None}
```

Comment:
Almost certainly we want to get the context first from `super().get_serializer_context()` and then update only the value for the `'request'` key.

Reply:
Yes, nice catch.